doc/api/modules.md

```diff
@@ -649,9 +649,11 @@ changes:
 * `request` {string} The module path to resolve.
 * `options` {Object}
   * `paths` {string[]} Paths to resolve module location from. If present, these
-    paths are used instead of the default resolution paths. Note that each of
-    these paths is used as a starting point for the module resolution algorithm,
-    meaning that the `node_modules` hierarchy is checked from this location.
+    paths are used instead of the default resolution paths, with the exception
+    of [GLOBAL_FOLDERS](#modules_loading_from_the_global_folders) like
```

Comment:
Since the link #modules_loading_from_the_global_folders is used twice in the document, it can be moved to bottom of the page as follows:

[GLOBAL_FOLDERS]: #modules_loading_from_the_global_folders